src/dlfcn.c

```diff
@@ -216,7 +216,7 @@ static void save_err_ptr_str( const void *ptr, DWORD dwMessageId )
     save_err_str( ptr_buf, dwMessageId );
 }
 
-static HMODULE MyGetModuleHandleFromAddress( void *addr )
+static HMODULE MyGetModuleHandleFromAddress( const void *addr )

```

Comment:
I do not think that it makes sense to update `void *` to `const void *` for every internal dlfcn-win32 function, because at the end is `const` removed, see line 240 where is `(LPCSTR) addr`.

Reply:
on the contrary, LPCSTR can be removed now